flink-contrib/flink-storm-compatibility/flink-storm-compatibility-examples/src/main/java/org/apache/flink/stormcompatibility/excamation/ExclamationWithStormBolt.java

```diff
@@ -88,9 +88,23 @@ public static void main(final String[] args) throws Exception {
 
 	private static class ExclamationMap implements MapFunction<String, String> {
 
+		private String exclamation;
+
+		public ExclamationMap() {
```

Comment:
Use `exclamationNum` as constructor parameter and build `exclamation` accordingly.